rules/opentitan/ci.bzl

```diff
@@ -0,0 +1,43 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@bazel_skylib//lib:sets.bzl", "sets")
+
+# List of execution environments that are always enabled in CI.
+_ALWAYS_RUN_IN_CI = [
```

Comment:
Instead of returning a list of environments to run CI, perhaps it should return the skip_for_ci list? In that case we don't need this list.

Reply:
Thanks for the suggestion, I will implement that.

Reply:
I have updated the code.